### ISSUE TITLE
Will the submission script written by `fair run` always be a `sh` file?

### ISSUE THREAD

**Issue by soniamitchell** [FAIRDataPipeline/rDataPipeline#170]
Currently, I've been assuming it's a `sh` file. But presumably if it's copying a script from `script_path`, then it could be an `R` file, for example? In which case, is the `script_path` in the working config something that's staying?

**Comment by richardreeve** [comment]
No, it's not set to be a `.sh` file - you can set the shell, so it could just be an R script if you set the shell to R. At the moment you should only need to "guess" the location of the working config file, because that will contain the path to the script in `run_metadata`. However, it's an argument to `initialise()`! Not ideal...

The idea is before Christmas(?) we rewrite this so we register the files in the registry and pass the `uuid`s of the object (or the file hashes) to the DPAPI rather than the folder, but then we need to know how to return the coderuns that were created... for the time being any hack that works is fine.

**Comment by richardreeve** [comment]
In particular `script_path` would become `script_uuid` probably?

**Comment by soniamitchell** [comment]
The code runs are still output to `coderuns.txt`?

Is the script still called `script` at least, whatever the extension might be?

**Comment by richardreeve** [comment]
For the time being everything stays the same. After we've submitted, we can start playing around with it again. For now, definitely `script.ext` and `coderuns.txt` so everyone is developing against a fixed target.

**Comment by richardreeve** [comment]
In fact for now `script.sh` and `coderuns.ext`...

**Comment by soniamitchell** [comment]
Is this a typo? or is there some benefit to coderuns having different extensions?

**Comment by richardreeve** [comment]
It's a typo, sorry - `coderuns.txt`!
